docs/_data/hiddenItems.yaml

```diff
@@ -0,0 +1,3 @@
+# NOTE: Add elements to this file if you don't want them to appear on
+# the "All Elements" page or under "Elements" in the side navigation.
+- rh-menu
```

Comment:
can we put this in repoStatus instead?

Reply:
Resolved in [ec17526](https://github.com/RedHat-UX/red-hat-design-system/pull/1988/commits/ec17526ca8888685fcb204e118d3c3b15116718b).

Also, I moved the `comingSoonItems.yaml` file to `repoStatus.yaml` since we seem to be trending that way.